src/site/content/en/blog/text-fragments/index.md

```diff
@@ -320,7 +320,7 @@ contains both an element fragment (`HTML1`), as well as a text fragment
 (`text=Give%20us%20feedback%20in%20our%20Product%20Forums.`):
 
 <figure class="w-figure">
-  <img src="text-feedback.png" alt="" class="w-screenshot" width="237">
+   <img src="text-feedback.png" alt="" class="w-screenshot" width="237">
```

Comment:
looks like this image was missed

Reply:
This was missed due to no alt tag present, I'll get this in another batch of uploads.